src/DataTool.php

```diff
@@ -181,7 +168,32 @@ public function generateData()
             }
         }
     }
-    
+
+    /**
+     * @param $field string - stores field name from fields
+     * @param $fieldDef array - stores data definition for proper field
+     */
+    public function generateFieldData($field, $fieldDef)
```

Comment:
As agreed via HipChat conversation let't not pass `$fieldDef` as an argument to this method, but rather derive it as `$this->fields[$field]`.